products/alp/main.pm

```diff
@@ -66,6 +66,11 @@ if (get_var('BOOT_HDD_IMAGE')) {
     load_selfinstall_boot_tests;
 }
 
+if (get_var('CONTAINER_RUNTIME') eq 'k3s') {
+    loadtest('containers/run_container_in_k3s');
+    return 1;
+}
```

Comment:
This should be done in `lib/main_containers.pm` which is loaded in this file on line `78`.

Reply:
This is the main that uses file. Following the clonation of alp/containers this main will be executed

Reply:
I don't really understand.

Reply:
`$distri/main.pm` imports functions from `lib/main_containers.pm`. As long as the test suite should be used solely for ALP, it is fine to keep it in `alp/main.pm`. Is it the case?

Reply:
yes, it is only for ALP

Reply:
The test has `container` in its name. How is that not part of containers then? I understand that now you're enabling it for ALP. But it can be also enabled elsewhere. Having some of the container tests scheduled one way and some other just increases complexity.

Reply:
But we can add 'loadtest('containers/run_container_in_k3s');' to main_container.pm if it's necessary, but rigth now there is not any thicket asking that

Reply:
Feel free to ignore the suggestion. I might play with this on the weekend.